CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -1410,4 +1410,26 @@ public static function civiMailingCreatedDateNull(CRM_Queue_TaskContext $ctx) {
     return TRUE;
   }
 
+  /**
+   * 'Related Cases' tab for organizations: Add enable/disable 'Related Cases' setting
+   * @return bool
+   */
+  public static function settingForRelatedCasesTab() {
```

Comment:
I don't think just adding the function to the upgrader will do anything. You need to add it to an upgrader function of the form `upgrade_4_7_x` . Just add it to upgrade_4_7_32 for now, and if there's any problem, core team will let us know. This would be the way to do it:

```php
  public function upgrade_4_7_32($rev) {
    ...
    $this->addTask('CRM-21829: Show Related Cases For Organizations', 'settingForRelatedCasesTab');
  }
```